packages/components/modal/src/ModalHeader/ModalHeader.tsx

```diff
@@ -40,6 +42,11 @@ export const ModalHeader = ({
     >
       <Subheading as="h2" isTruncated marginBottom="none">
         {title}
+        {subtitle && (
+          <Text marginLeft="spacingXs" color="gray700">
```

Comment:
```suggestion
          <Text marginLeft="spacingXs" fontColor="gray700">
```

Reply:
We should remove the `color` prop, it's so confusing

Reply:
https://github.com/contentful/forma-36/issues/2339 😄